utils/model_utils.py

```diff
@@ -60,6 +62,13 @@ def get_model_file_name(id: str, model_name: str, timestamp: str) -> str:
 
 	return model_file_name
 
+def get_dataset_file_name(id: str, model_name: str, timestamp: str, prefix: str) -> str:
+	if type(model_name) != str:
+		raise ValueError("Model name must be string")
+
+	model_file_name = f"{id}-{model_name}-{timestamp}-{prefix}.npy"
```

Comment:
Small change to variable naming

```suggestion
	dataset_file_name = f"{id}-{model_name}-{timestamp}-{prefix}.npy"
```